src/patternfly/components/Progress/progress.hbs

```diff
@@ -21,8 +21,12 @@
   {{#if progress--attribute}}
     {{{progress--attribute}}}
   {{/if}}>
-  {{> progress-description}}
-  {{> progress-status}}
+  {{#if progress__description}}
+    {{> progress-description}}
+  {{/if}}
+  {{#unless progress--no-measure}}
+    {{> progress-status}}
+  {{/unless}}
```

Comment:
I think we only want to remove the status element if there is no measure and no status icon. This update removes the icon from this example, which I don't think we want to do. https://patternfly-pr-5387.surge.sh/components/progress#failure-without-measure

This is the current v4 example that has an empty `__status` element - https://pf4.patternfly.org/components/progress#without-measure

<img width="903" alt="Screenshot 2023-02-16 at 3 24 19 PM" src="https://user-images.githubusercontent.com/35148959/219490434-f77adedc-0e52-416b-a00e-8ba50546f17c.png">

Another small thing is that there is a margin on the status icon when there is no measure. What might be better is to turn `__status` into `display: flex; justify-content: flex-end; align-items: flex-start; gap: var(--whatever-spacer-that-margin-uses)`. That will also remove any space created by whitespace between the measure and status icon, too, and let the gap serve as the only source of space between those elements.

Reply:
Added the styling to __status, and also removed the `status-icon__MarginLeft` var since this is a breaking change issue and it would no longer serve any purpose.

The conditional rendering should be working as intended now.